GNNLux/test/layers/conv_tests.jl

```diff
@@ -94,6 +97,28 @@
         test_lux_layer(rng, l, g, x, sizey=(out_dims,g.num_nodes), container=true)
     end
 
+     
+
+    @testset "NNConv" begin
+        n_in = 3
+        n_in_edge = 10
+        n_out = 5
+        
+        s = [1,1,2,3]
+        t = [2,3,1,1]
+        g2 = GNNGraph(s, t)
+        
+        nn = Dense(n_in_edge => n_out * n_in)
+        l = NNConv(n_in => n_out, nn, tanh, aggr = +)
+        x = randn(Float32, n_in, g2.num_nodes)
+        e = randn(Float32, n_in_edge, g2.num_edges)
+        #y = l(g, x, e) # just to see if it runs without an error
+        #edim = 10
+        #nn = Dense(edim, in_dims * out_dims)        
+        #l = NNConv(in_dims => out_dims, nn, tanh, aggr = +)
+        test_lux_layer(rng, l, g2, x, sizey=(n_out, g2.num_nodes), container=true, edge_weight=e) 
+    end          
```

Comment:
> Can you paste the code that errors?

i used the code that you gave for tests (in the test file) 
it errors while testing

Reply:
as I said, can you translate this
```julia
using GraphNeuralNetworks, Flux

n_in = 3
n_in_edge = 10
n_out = 5

s = [1,1,2,3]
t = [2,3,1,1]
g = GNNGraph(s, t)

nn = Dense(n_in_edge => n_out * n_in)
l = NNConv(n_in => n_out, nn, tanh, bias = true, aggr = +)
x = randn(Float32, n_in, g.num_nodes)
e = randn(Float32, n_in_edge, g.num_edges)
y = l(g, x, e)  
```
to the corresponding GNNLux code and see what happens?

Reply:
tried to run locally today but seems like there is an update to some package that leads to this error:
`ERROR: LoadError: Failed to precompile MLDataDevices`

while importing GNNlib

Reply:
> Try to run the corresponding LuxGNN version 
```
        n_in = 3
        n_in_edge = 10
        n_out = 5
        
        s = [1,1,2,3]
        t = [2,3,1,1]
        g2 = GNNGraph(s, t)
        
        nn = Dense(n_in_edge => n_out * n_in)
        l = NNConv(n_in => n_out, nn, tanh, aggr = +)
        x = randn(Float32, n_in, g2.num_nodes)
        e = randn(Float32, n_in_edge, g2.num_edges)

        ps = LuxCore.initialparameters(rng, l)
        st = LuxCore.initialstates(rng, l)
        
        y = l(g, x, e, ps, st)
```

added this to the test file, there we can see what errors

Reply:
error:
```
  Got exception outside of a @test
  AssertionError: Got 3 as last dimension size instead of num_nodes=10
```

Reply:
Any suggestions?

Reply:
getting the same dimension error